lib/std/json/static.zig

```diff
@@ -42,6 +43,12 @@ pub const ParseOptions = struct {
     /// The default with a `*std.json.Reader` input is `.alloc_always`.
     /// Ignored for `parseFromValue` and `parseFromValueLeaky`.
     allocate: ?AllocWhen = null,
+
+    // Specifies whether or not integers and floats should be left as their original string representations,
+    // or parsed into their 'std.json.Value.integer' or 'std.json.Value.float' representations.
+    // Exact precision floats and integers are both represented as a 'std.json.Value.number_string';
+    exact_precision_integers: bool = false,
+    exact_precision_floats: bool = false,
```

Comment:
I think the `exact_precision` naming might be a bit confusing since its not clear that no parsing will happen.  Perhaps `parse_integers` and `parse_floats` might be more clear and direct?

Also, both fields should have a doc comment which tooling can make use of.  Maybe something like this (note the triple slashes):

```zig
/// When true integers will be parsed and stored as `std.json.Value.integer`s.  When false, integers will be stored as `std.json.Value.number_string` without being parsed.
parse_integers: bool = true,
/// When true floats will be parsed and stored as `std.json.Value.float`s.  When false, floats will be stored as `std.json.Value.number_string`s without being parsed.
parse_floats: bool = true,
```

Reply:
It's definitely more clear and to point to use the parse_**** naming scheme. I also went ahead and copied your doc comments in, they're in the proper format and more what I'd want to see when reading the documentation.